src/main/java/de/retest/recheck/printer/MetadataDifferencePrinter.java

```diff
@@ -33,19 +33,22 @@
 
 	@Override
 	public String toString( final MetadataDifference difference, final String indent ) {
-		final String prefix = "Metadata Differences:";
-		return indent + prefix + printDifferences( difference, indent + "\t" );
+		final String differences = printDifferences( difference, indent + "\t" );
+		return differences.isEmpty() ? "" : indent + "Metadata Differences:" + differences;
```

Comment:
Doesn't this then have a empty line in TestPrinter? Since currently printers are supposed to always "print" a line, since all lines are just stupidly concatenated...

Could you write a test for that?